src/lib/components/NewLayeredMap/NewLayeredMap.jsx

```diff
@@ -7,7 +7,7 @@ import './layers/L.imageWebGLOverlay';
 import './layers/L.tileWebGLLayer';
 
 // Components
-import Controls from './components/Controls';
+import Controls from './components/controls/Controls';
```

Comment:
Call the folder for **Controls** and then have a index.js inside of that folder which exports the Controls.js.
```
export { default } from './Controls';
```
With this, the end result will be the following import:
```
import Controls from './components/Controls';
```
...which is more convenient than `./components/controls/Controls` 😛